tempto-core/src/test/groovy/io/prestodb/tempto/internal/ssh/JSchSshClientTest.groovy

```diff
@@ -141,12 +143,15 @@ class JSchSshClientTest
 
     private void setupAuthentication()
     {
-        sshd.setUserAuthFactories([new UserAuthPassword.Factory(), new UserAuthPublicKey.Factory()])
+        List<UserAuthFactory> userAuthFactories = new ArrayList<>();
+        userAuthFactories.add(new UserAuthPasswordFactory())
+        userAuthFactories.add(new UserAuthPublicKeyFactory())
+        sshd.setUserAuthFactories(userAuthFactories)
 
         sshd.setPasswordAuthenticator(new PasswordAuthenticator() {
             boolean authenticate(String username, String password, ServerSession session)
             {
-                return USER.equals(username) && PASSWORD.equals(password);
+                return USER == username && PASSWORD == password;
```

Comment:
Should use `.equals` to compare content ?

Reply:
I think in groovy it is different than Java. IntelliJ suggested the change. I can revert it though

Reply:
https://www.groovy-lang.org/differences.html#_behaviour_of